adaptor/index.html

```diff
@@ -183,7 +183,9 @@
         </tr>
       </tbody>
     </table>
-<br>
+    <p>
+      See the <a href="docs/KnownIssues.html">Known Issues</a> for information about an installer error on Windows Server 2012 R2 with Java 8.
```

Comment:
I wanted to call out the Known Issues page as a generic destination, though we currently haven't put anything else there. I also don't know for sure about the exact combination, so I wanted to be a little vague. Basically, if someone has an installer error, I'd like them to try this link. We are trying to narrow down the error, which might also be specific instead to Java 8 Update 60 or later, so I would change the string accordingly. We may even fix the issue, there's an InstallAnywhere hotfix that looks promising.

I like your "steps to run" language and the Note. What about:

**Note:** See the Known Issues for steps to resolve an installer error on Windows Server 2012 R2 with Java 8.

John L